src/temaWeek7Comparator/Persoane.java

```diff
@@ -3,7 +3,6 @@
  * Implemented 2 comparators on a treeSet of persons:NameComparator and AgeComparator
  */
 
-
 import java.util.*;
 
 public class Persoane {
```

Comment:
Also, I would have placed the comparators in their own classes.
Please note that we shouldn't use the casts here. We are using generics so the casts to Persoane are not needed anymore.